4001-5000/LIT4975Dersan.xml

```diff
@@ -42,19 +47,29 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
             <textClass>
                 <keywords>
                     <term key="Homily"/>
-                    <term key="ChristianLiterature"/>
+                    <term key="Translation"/>
+                   <term key="ChristianLiterature"/>
                 </keywords>
             </textClass>
             <langUsage><language ident="en">English</language><language ident="gez">Gǝʿǝz</language></langUsage>
         </profileDesc>
-        <revisionDesc>
```

Comment:
the file does not validate because part of revisionDesc for some reason has been removed, while the closing tag stayed